packages/entities/entities-plugins/src/components/select/PluginSelectCard.vue

```diff
@@ -7,12 +7,11 @@
       class="plugin-card-content"
       :class="{ disabled: isDisabled }"
       :data-testid="`${plugin.id}-card`"
-      has-hover
       @click="isDisabled || isCustomPlugin ? undefined : handleClick()"
     >
       <template
         v-if="isCustomPlugin"
-        #statusHat
+        #footer
```

Comment:
The overflow button appeared on the top-left corner in custom plugin cards–is this expected?

<img width="772" alt="Screenshot 2023-12-04 at 15 31 34" src="https://github.com/Kong/public-ui-components/assets/5277268/065a34e5-9fc4-4a0c-9da3-ec4424dd48c0">

Reply:
Ah good catch! Fixed it here but also added a fix to KCard to prevent this from happening